### ISSUE TITLE
Do not "simplify" returned route

### ISSUE THREAD

**Issue by HarelM** [perliedman/geojson-path-finder#87]
Thanks for this awesome tool!
I'm looking into integrating it in order to allow small offline routing using vector data that is on the device.
Beside the issue I opened recently (#86), the following is also problematic in terms of results:
Green is that start point, red is the end point, and the blue line is the routing.
As can be seen the route itself is not using the original data points but probably the post process points (if I need to guess).
![image](https://user-images.githubusercontent.com/3269297/227458579-ab2780f7-9eaa-430c-a49e-23c1721ad02e.png)
An expected results would be to get all the points on the original data set.

**Comment by HarelM** [comment]
Yup, Thanks! I was planning on doing so. It does solve the problem.
I guess my main concern is how to create a route between any two points - i.e. #86

**Comment by perliedman** [comment]
I know we're more than one and a half year after this was reported, sorry about that, but FWIW...

Those results do look a little strange, although it's hard to tell since there's no scale. The default tolerance is `1e-5` which, assuming WGS84 GeoJSON input, means precision should be about 1 meter. So that level of coarseness isn't expected, I think. Looking at for example the lib's demo app, you can see that the simplified results still are very close to the source data:

<img width="355" height="302" alt="Image" src="https://github.com/user-attachments/assets/fde44747-3fb0-488b-bcc4-5d4e3ba10fac" />

If this continues to be an issue, please provide an example so I can have a look.

**Comment by perliedman** [comment]
*Two* and a half years after reported, sorry... 😅

**Comment by HarelM** [comment]
Yeah, this is definitely long overdue and I don't fully remember what I did to overcome all the issues I had.
The following issue adds some details to the process I did to make the feature work: 
https://github.com/IsraelHikingMap/Site/issues/1636
The following is the production code and initialization of this library: 
https://github.com/IsraelHikingMap/Site/blob/9c7c5072d9e58bfd4030cec33cd9dd79946d67c7/IsraelHiking.Web/src/application/services/routing.provider.ts#L111

In the above issue you can find a stackblitz code that I used to debug my issues and see the what happens when you change coordinates and tolerance (I hope it's still functional).
